src/TextLogging/src/Logger.cpp

```diff
@@ -0,0 +1,34 @@
+/**
+ * @file Logger.cpp
+ * @authors Giulio Romualdi
+ * @copyright 2021 Istituto Italiano di Tecnologia (IIT). This software may be modified and
+ * distributed under the terms of the GNU Lesser General Public License v2.1 or any later version.
+ */
+
+#include <spdlog/sinks/stdout_color_sinks.h>
+
+#include <BipedalLocomotion/TextLogging/Logger.h>
+#include <iostream>
+namespace BipedalLocomotion
+{
+
+TextLogging::Logger* const log()
+{
+    auto logger = spdlog::get("console");
```

Comment:
I don't know if it make sense to use a blf-specific name for this, especially given that we modify it (set level, set pattern) so it it may be better to avoid conflicts with other libraries that already may use spdlog "console"?